driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionTest.java

```diff
@@ -196,16 +195,23 @@ public void setUp() {
             }
         }
 
-        Map<String, Map<String, Object>> kmsProvidersMap = new HashMap<String, Map<String, Object>>();
-
+        Map<String, Map<String, Object>> kmsProvidersMap = new HashMap<>();
         for (String kmsProviderKey : kmsProviders.keySet()) {
             BsonDocument kmsProviderOptions = kmsProviders.get(kmsProviderKey).asDocument();
-            Map<String, Object> kmsProviderMap = new HashMap<String, Object>();
-
+            Map<String, Object> kmsProviderMap = new HashMap<>();
             if (kmsProviderKey.equals("aws")) {
                 kmsProviderMap.put("accessKeyId", System.getProperty("org.mongodb.test.awsAccessKeyId"));
                 kmsProviderMap.put("secretAccessKey", System.getProperty("org.mongodb.test.awsSecretAccessKey"));
                 kmsProvidersMap.put("aws", kmsProviderMap);
+            } else if (kmsProviderKey.equals("azure")) {
```

Comment:
Not necessary to change it if you don't want, but I was recently reminded that as of Java 7 you can switch on strings.